apps/sms/views/shared/js/startup.js

```diff
@@ -113,21 +119,18 @@ var Startup = exports.Startup = {
 
       // Init UI Managers
       TimeHeaders.init();
-      ConversationView.init();
-      MessagingClient.init(App.instanceId);
-      MozMobileConnectionsClient.init(App.instanceId);
       Information.initDefaultViews();
 
+      return ConversationView.init();
```

Comment:
> I think this change make sense, but I can't figure out a better way to verify this in unit test(we can stub the Conversation since it's lazy loaded).

Mmm, but in unit test it's not actually lazy loaded since with use Mock for lazy loader. Can't we just load ConversationView mock in unit test, stub ```init``` method with our ```new Promise((resolve) => resolveConversationInit = resolve)```, and then call ```resolveConversationInit``` in unit test and verify that ```Navigation.setReady``` is called after that?